apps/api/src/controllers/v1/types.ts

```diff
@@ -195,6 +196,16 @@ const baseScrapeOptions = z
     extract: extractOptions.optional(),
     // New
     jsonOptions: extractOptions.optional(),
+    changeTrackingOptions: z
+      .object({
+        prompt: z.string().optional(),
+        systemPrompt: z.string().optional(),
+        schema: z.any().optional(),
+        temperature: z.number().optional().default(0),
+        mode: z.enum(["llm"]).optional().default("llm"),
```

Comment:
is this a leftover fropm smth? or intentional

Reply:
Fixed!!